wrappers/python/anoncreds/bindings.py

```diff
@@ -964,3 +1001,257 @@ def create_or_update_revocation_state(
         byref(updated_rev_state),
     )
     return updated_rev_state
+
+
+def create_w3c_credential_offer(
+    schema_id: str, cred_def_id: str, key_proof: ObjectHandle
+) -> ObjectHandle:
+    cred_offer = ObjectHandle()
+    do_call(
+        "anoncreds_create_w3c_credential_offer",
+        encode_str(schema_id),
+        encode_str(cred_def_id),
+        key_proof,
+        byref(cred_offer),
+    )
+    return cred_offer
+
+
+def create_w3c_credential_request(
+    entropy: Optional[str],
+    prover_did: Optional[str],
+    cred_def: ObjectHandle,
+    link_secret: str,
+    link_secret_id: str,
+    cred_offer: ObjectHandle,
+) -> Tuple[ObjectHandle, ObjectHandle]:
+    cred_req, cred_req_metadata = ObjectHandle(), ObjectHandle()
+    do_call(
+        "anoncreds_create_w3c_credential_request",
+        encode_str(entropy),
+        encode_str(prover_did),
+        cred_def,
+        encode_str(link_secret),
+        encode_str(link_secret_id),
+        cred_offer,
+        byref(cred_req),
+        byref(cred_req_metadata),
+    )
+    return (cred_req, cred_req_metadata)
+
+
+def create_w3c_credential(
+    cred_def: ObjectHandle,
+    cred_def_private: ObjectHandle,
+    cred_offer: ObjectHandle,
+    cred_request: ObjectHandle,
+    attr_raw_values: Mapping[str, str],
+    revocation_config: Optional[RevocationConfig],
+    encoding: Optional[str],
```

Comment:
@Artemkaaas why change the signature of the attributes?  the original create_credential is

```
 attr_raw_values: Mapping[str, str],
    attr_enc_values: Optional[Mapping[str, str]],
    revocation_config: Optional[RevocationConfig],
    ```
    
    maybe good to keep the signature the same if we can?

Reply:
It was decided to move encoding of credential values to the library